src/otx/core/data/dataset/visual_prompting.py

```diff
@@ -231,22 +246,24 @@ def _get_item_impl(self, index: int) -> ZeroShotVisualPromptingDataEntity | None
                     )
                     gt_prompts.append(point)
 
-                gt_labels.append(annotation.label)
+                gt_labels["prompts"].append(annotation.label)
+                gt_labels["polygons"].append(annotation.label)
+                gt_labels["masks"].append(annotation.label)
```

Comment:
It's just a question. If same list is assigned to prompts, polygons and masks, isn't it possible to keep single list?

Reply:
The purpose of this implementation is to distinguish various inputs' labels.
Not implemented yet, but if datumaro boundingbox or point is only given wihout polygon or mask, labels of polygon and mask will not be set.
It is the implementation for zero shot learning that can get dynamic input types.